public/static/text/kn-mn.json

```diff
@@ -10588,6 +10588,10 @@
                         "type": "paragraph",
                         "text": "**සාතං අසාතන්ති යමාහු ලොකෙ**ති - ‘ සාත’න්ති සුඛා ච වෙදනා ඉට්ඨඤ්ච වත්ථු,{4} **අසාත**න්ති දුක්ඛා ච වෙදනා අනිට්ඨඤ්ච වත්ථු; **යමාහු ලොකෙ**ති යං ආහංසු යං කථෙන්ති යං භණන්ති යං දීපයන්ති යං වොහරන්තීති - ‘ සාතං අසාතන්ති යමාහු ලොකෙ.’"
                     },
+                    {
+                        "type": "paragraph",
+                        "text": "**තමූපනිස්සාය පහොති ඡන්දො**ති. සාතාසාතං නිස්සාය, සුඛදුක්ඛං නිස්සාය, සොමනස්සදොමනස්සං නිස්සාය, ඉට්ඨානිට්ඨං නිස්සාය, අනුනයපටිඝං නිස්සාය ඡන්දො පහොති පභවති ජායති සඤ්ජායති නිබ්බත්තති අභිනිබ්බත්තතීති - ‘ තමූපනිස්සාය පහොති ඡන්දො.’{*}"
```

Comment:
there is a space before තමූපනිස්සාය which can be removed. otherwise looks great

Reply:
Seems like there are few additional spaces before quotation marks like: `‘ සාත’න‍්ති`,  `‘ සාතං `, `‘ රූපෙසූ’ති`, `රූපෙසු දිස්වා විභවං භවඤ්ච. ‘` (See above and above lines). Therefore, I kept space also here (thought it was a convention). May I remove those additional spaces?
![image](https://user-images.githubusercontent.com/34299693/109893361-89547000-7cb1-11eb-866c-d5eb6f80bac8.png)

Reply:
Got it. yes those spaces can be removed